store/cafs/src/addFilesFromDir.ts

```diff
@@ -41,7 +41,7 @@ async function _retrieveFileIntegrities (
 ) {
   try {
     const files = await fs.readdir(currDir)
-    await Promise.all(files.map(async (file) => {
+    await Promise.allSettled(files.map(async (file) => {
       const fullPath = path.join(currDir, file)
       const stat = await fs.stat(fullPath)
```

Comment:
I guess the right way would be add a try/catch here. In the catch, if the error code is `ENOENT`, check if the file was a symlink, if so, ignore the error.